doc/development/deprecations.rst

```diff
@@ -6,6 +6,24 @@ Deprecations
 Pending deprecations
 --------------------
 
+* The `grouping_type` and `grouping_method` arguments of `qchem.molecular_hamiltonian()` are deprecated.
+
+  - Deprecated in v0.24
+  - Will be removed in v0.31
```

Comment:
Is it really deprecated in v0.24?

Reply:
I don't remember exactly, I think it might have been a typo